{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/templates/base.html

```diff
@@ -61,7 +61,7 @@
           {% block content_below %}{% endblock %}
         </div>
 
-        {% include 'base/_footer.html' %}
+        {% include 'footer/_footer.html' %}
```

Comment:
I'm not sure if this change is to use the footer app or not. If it is, you should probably delete the base/_footer file.